dev-python/urllib3/urllib3-1.26.4.ebuild

```diff
@@ -58,6 +62,32 @@ python_test() {
 	local deselect=(
 		# TODO?
 		test/with_dummyserver/test_socketlevel.py::TestSocketClosing::test_timeout_errors_cause_retries
+		# Fail because they rely on warnings and there are new deprecation warnings in 3.10
```

Comment:
```suggestion
	)
	[[ ${EPYTHON} == python3.10 ]] && deselect+=(
		# Fail because they rely on warnings and there are new deprecation warnings in 3.10
```

Reply:
Amended